src/command.ts

```diff
@@ -17,7 +17,7 @@ const defaultOptions: SnapshotOptions = {
   screenshotsFolder,
   isUpdateSnapshots,
   isSnapshotDebug,
-  specFileName: Cypress.spec.name,
+  specFileRelativeToRoot: Cypress.spec.relative,
```

Comment:
Do you think this behaviour should be optional? I'm wondering if it could surprise people that the structure beneath `snapshots` doesn't match `screenshots` (even though I can't imagine anyone wanting the new default Cypress behaviour!)

Reply:
Sounds good. Simple is better. People can always feedback if they have a problem with it.